src/transformers/models/efficientformer/configuration_efficientformer.py

```diff
@@ -0,0 +1,185 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" EfficientFormer model configuration"""
+
+from collections import OrderedDict
+from typing import List, Mapping
+
+from packaging import version
+
+from ...configuration_utils import PretrainedConfig
+from ...onnx import OnnxConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+EFFICIENTFORMER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "efficientformer-l1": "https://huggingface.co/efficientformer-l1/resolve/main/config.json",
```

Comment:
The repo name and path should include the repo owner user / organization. You can upload the repo under your own username and we can transfer it to the huggingface organization or Snap Research if they create an organization on the hub:
```suggestion
    "huggingface/efficientformer-l1": "https://huggingface.co/huggingface/efficientformer-l1/resolve/main/config.json",
```